app/helpers/service_provider_helper.rb

```diff
@@ -143,6 +143,7 @@ def map_config_attributes(sp_hash, sp)
       'default_aal'=> sp_hash['default_aal'],
       'attribute_bundle' => sp_hash['attribute_bundle'],
       'protocol' => sp_hash['protocol'],
+      'push_notification_url' => sp_hash['push_notification_url'],
```

Comment:
**question:** mind adding a test for this too? E.g. create an SP with a `push_notification_url` set, visit the page, verify that it's in the appropriate place on the page (since it will show up twice)? Thanks!

Reply:
Yeah I can add that!